Makefile.js

```diff
@@ -853,6 +884,12 @@ target.checkRuleFiles = function() {
                 console.error("Missing id in the doc page's title of rule %s", basename);
                 errors++;
             }
+
+            // check for proper doc headers
+            if (!hasKnownHeaders(basename)) {
```

Comment:
It looks like this function doesn’t accept any arguments?

Reply:
Originally I had the function do the logging and it used this argument for that, but I refactored it to do the logging outside the function here, to bring it in line with the others checks and I forgot to remove the argument here. Will fix this when I make the other changes.